spring-aot/src/main/java/org/springframework/aot/factories/IgnoredFactoriesCodeContributor.java

```diff
@@ -52,12 +53,21 @@
 			"org.springframework.boot.test.autoconfigure.filter.TypeExcludeFiltersContextCustomizerFactory",
 			"org.springframework.boot.test.context.ImportsContextCustomizerFactory");
 
+	private static final Predicate<SpringFactory> IGNORED_TEST_EXECUTION_LISTENERS = factoryEntry(
+			"org.springframework.test.context.TestExecutionListener",
```

Comment:
Could you elaborate on why `org.springframework.test.context.TestExecutionListener` exclusion is needed? Sounds like a general purpose one that could likely break regular tests if removed.

Reply:
The first parameter of `factoryEntry` is the key of the entry in `spring.factories`. This change only exclude the two mockito listeners, nothing else.

Reply:
Good point, that's great then.